modules/apps/bean-portlet/bean-portlet-cid-extension-sample-test/src/main/webapp/WEB-INF/views/view.xhtml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+
+<f:view
+	xmlns="http://www.w3.org/1999/xhtml"
+	xmlns:f="http://xmlns.jcp.org/jsf/core"
+	xmlns:h="http://xmlns.jcp.org/jsf/html"
+>
+	
+	<h:body>
+
```

Comment:
@Kvale1733, please remove redundant TAB in the line 8. It seems SF didn't detect it.